wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```
@@ -1927,7 +1944,7 @@ public static function sendCampOutcomeAckEmailAfter5Days($collectionCamp) {
   /**
    *
    */
-  public static function getCampOutcomeAckEmailAfter5Days($attendeeName, $campAddress, $campDate, $totalAmount, $materialGenerated, $uniqueContributors, $campRating, $fundsGenerated, $campId) {
+  public static function getCampOutcomeAckEmailAfter5Days($attendeeName, $campAddress, $campDate, $totalAmount, $materialGeneratedHtml, $uniqueContributors, $campRating, $fundsGenerated, $campId) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$totalAmount'. (undefined)

(UnusedFormalParameter)